examples/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+# Example usage of exporter in use
+
+# In this case, it depends on the proper ceph keys in the correct place.
+
+version: '2'
+
+services:
+  prometheus:
+    image: prom/prometheus:v1.6.1
+    volumes:
+        - './prometheus.yml:/etc/prometheus/prometheus.yml'
```

Comment:
The single quotes should be redundant here, we can remove them for everything but ports (since essentially everything is a string in YAML unless it looks like a number or yes/no/true/false/on/off) .